config/crds/hive.openshift.io_selectorsyncidentityproviders.yaml

```diff
@@ -495,13 +495,28 @@ spec:
                               items:
                                 type: string
                               type: array
+                              x-kubernetes-list-type: atomic
+                            groups:
+                              description: groups is the list of claims value of which
+                                should be used to synchronize groups from the OIDC
+                                provider to OpenShift for the user. If multiple claims
+                                are specified, the first one with a non-empty value
+                                is used.
+                              items:
+                                description: OpenIDClaim represents a claim retrieved
+                                  from an OpenID provider's tokens or userInfo responses
+                                minLength: 1
+                                type: string
+                              type: array
+                              x-kubernetes-list-type: atomic
```

Comment:
What are these selectorsyncset changes all about?

Reply:
These crd changes were generated after bumping mods in the api submodule.